plugin/src/test/java/org/jenkinsci/plugins/structs/describable/second/SharedName.java

```diff
@@ -26,10 +26,11 @@
 
 import hudson.Extension;
 import hudson.model.Descriptor;
-import org.jenkinsci.Symbol;
 import org.jenkinsci.plugins.structs.describable.AbstractSharedName;
 import org.kohsuke.stapler.DataBoundConstructor;
 
+import javax.annotation.Nonnull;
+
```

Comment:
The NonNull annotation from spotbugs is preferred over the javax annotation
```suggestion
import javax.annotation.Nonnull;

```